src/ProgramBlockEditor.js

```diff
@@ -41,6 +41,7 @@ type ProgramBlockEditorProps = {
 
 type ProgramBlockEditorState = {
     showConfirmDeleteAll: boolean,
+    showShareComplete: boolean,
```

Comment:
It looks like the changes in `ProgramBlockEditor.js` are left over from previous work and can be reverted?

Reply:
I reviewed the changes, what's left now seems appropriate (the new state variable).